packages/react/src/components/Slider/Slider.test.tsx

```diff
@@ -1,9 +1,6 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import * as ReactTestUtils from 'react-dom/test-utils';
-import { mount, ReactWrapper } from 'enzyme';
-import { resetIds, KeyCodes } from '@fluentui/utilities';
-import { create } from '@fluentui/utilities/lib/test';
+import { resetIds, KeyCodes } from '../../Utilities';
+import { fireEvent, render } from '@testing-library/react';
```

Comment:
This is going to be in the testing guidance, but it's generally recommended to use userEvent rather than fireEvent, since it more realistically simulates the full sequence of events triggered by user interaction. https://www.npmjs.com/package/@testing-library/user-event

Reply:
I can port some of the mouseclick behaviors to use `userEvent` but there are some tests (such as checking if `onChange` triggers on a `mousedown` click while `onChanged` does not) where using `fireEvent.mousedown` was needed. Also, for the keyboard tests, using `userEvent.type(element, '{arrowdown}')` syntax to simulate a keydown was throwing the error below which is why I settled on using `fireEvent.keydown(element, {keyCode})`:

![image](https://user-images.githubusercontent.com/8649804/156454825-e05766d4-7982-4dbb-a041-b0cb5b59c61b.png)

Reply:
Oh interesting, I bet it's because it tries to do a click first. The docs mention there's a `skipClick` option--what happens if you use that?
> type will click the element before typing. To disable this, set the skipClick option to true.

Reply:
Nevermind on this error, just needed to change my approach on testing it